backend/laundry/models.py

```diff
@@ -5,18 +5,18 @@
 
 
 class LaundryRoom(models.Model):
```

Comment:
Can you explain why previously we had a hall_id and why we have a room_id now? 

What is the difference between them? 

Depending on this answer, I may be hesitant to delete UUID.

Reply:
hall_id was just a generic id/pkey (0, 1, 2...) on our end. UUID was a specific id that was used to access the room from an end point.

The new API has a more relational structure, where there's a one to many between locations and rooms. To keep track of this, we need to store both.